media/server/ipc/include/SessionManagementServer.h

```diff
@@ -58,7 +58,9 @@ class SessionManagementServer : public ISessionManagementServer
     SessionManagementServer &operator=(const SessionManagementServer &) = delete;
     SessionManagementServer &operator=(SessionManagementServer &&) = delete;
 
-    bool initialize(const std::string &socketName, unsigned int socketPermissions) override;
+    bool initialize(std::unique_ptr<firebolt::rialto::common::ILinuxWrapper> &linuxWrapper,
```

Comment:
In most of cases in rialto wrappers are passed in class constructor. Please move it there to make code more unified.